build.gradle.kts

```diff
@@ -155,7 +155,7 @@ dependencies {
     aar("com.google.firebase:firebase-config:21.6.0")
     aar("com.google.firebase:firebase-installations:17.2.0")
     // extracted aar dependencies
-    api(fileTree(mapOf("dir" to "build/jar", "include" to listOf("*.jar"))))
+    api(fileTree(mapOf("dir" to "build/jar", "include" to listOf("*.jar"), "exclude" to listOf("lifecycle-*"))))
```

Comment:
this excludes all of the following
![image](https://github.com/GitLiveApp/firebase-java-sdk/assets/1386607/77bdeed8-1045-4e3c-8461-dec026af3a22)
does it not break when not using compose?

Reply:
That aar break in compose. so need to exclude. And I also include androidx.lifecycle 2.8.0-rc01 it's fine in my project.

Reply:
My project is multiplatform project. So doesn't include `lifecycle-livedata`, `lifecycle-livedata-core`. But has `lifecycle-runtime`.

Do you think need to add lifecycle-runtime dependency?

Reply:
we can't be sure without finding out exactly where it is used in the android sdk source code, we can leave it out for now but lets remove this workaround once the dependencies in the aars are updated to the required version